cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectApiTest.java

```diff
@@ -96,7 +98,22 @@ public void after() {
     public void test(VertxTestContext context) throws InterruptedException {
         KafkaConnectApi client = new KafkaConnectApiImpl(vertx);
         CountDownLatch async = new CountDownLatch(1);
-        client.list("localhost", PORT)
+        client.listConnectorPlugins("localhost", PORT)
+            .compose(connectorPlugins -> {
+                assertEquals(connectorPlugins.size(), 2);
+
+                ConnectorPlugin fileSink = connectorPlugins.stream().filter(connector -> "org.apache.kafka.connect.file.FileStreamSinkConnector".equals(connector.getConnectorClass())).findFirst().orElse(null);
+                assertNotNull(fileSink);
+                assertEquals(fileSink.getType(), "sink");
+                assertEquals(fileSink.getVersion(), "2.3.1");
+
+                ConnectorPlugin fileSource = connectorPlugins.stream().filter(connector -> "org.apache.kafka.connect.file.FileStreamSourceConnector".equals(connector.getConnectorClass())).findFirst().orElse(null);
+                assertNotNull(fileSource);
+                assertEquals(fileSource.getType(), "source");
+                assertEquals(fileSource.getVersion(), "2.3.1");
```

Comment:
So we really want to assert on the exact version number? We'll end up having to do this every time we change the version of Kafka being used to run the test. I think it's enough to test non-emptiness.